### PR TITLE
Update documentation in readthedocs

### PR DESCRIPTION
**Pull request by aeng**
Update documentation in readthedocs

https://bugzilla.redhat.com/show_bug.cgi?id=1204982

### CODE REVIEW

File: docs/user-guide/account/account-activate.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#Activate account
+
+After signing up with Zanata, you will need to activate the account before performing any task.
```
Comment (djansen-redhat) [inline docs/user-guide/account/account-activate.md]: *tasks.
Or, to that matter, being able to log in at all...

File: docs/user-guide/account/account-activate.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+To activate your account:
+
+ 1. Check your email used for signup in Zanata. You should received an email with subject `Zanata account activation`.
```
Comment (djansen-redhat) [inline docs/user-guide/account/account-activate.md]: You should *have* received

File: docs/user-guide/account/account-activate.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+ 1. If you didn't received any email from Zanata, see [Resend activation email](user-guide/account/account-resend-activation).
```
Comment (djansen-redhat) [inline docs/user-guide/account/account-activate.md]: didn't *receive an* email

File: docs/user-guide/account/account-activate.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+ 1. Click the `Account activation link` in your email or copy and paste the URL provided into your browser.
+ 1. You will be redirect to Zanata with your account activated. You can now login with your details.
```
Comment (djansen-redhat) [inline docs/user-guide/account/account-activate.md]: be *redirected*

File: docs/user-guide/account/account-activate.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+ 
+<img alt="Account activated" src="images/account-activated.png" />
+
+*Note: The activation URL will expired after 24 hours.*
```
Comment (djansen-redhat) [inline docs/user-guide/account/account-activate.md]: will *expire*

File: docs/user-guide/account/account-resend-activation.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+# Resend activation email
+
+If you didn't received any email after signing up or entered wrong email address during signing up, you can always request to resend the account activation email or update your email address.
```
Comment (djansen-redhat) [inline docs/user-guide/account/account-resend-activation.md]: didn't *receive an*

File: docs/user-guide/account/account-resend-activation.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+ 
+ 1. Login into Zanata with your username and password that you used when signup.
```
Comment (djansen-redhat) [inline docs/user-guide/account/account-resend-activation.md]: *Log in to* ... when *signing up*

File: docs/user-guide/account/account-resend-activation.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
+# Resend activation email
+
+If you didn't received any email after signing up or entered wrong email address during signing up, you can always request to resend the account activation email or update your email address.
+ 
+ 1. Login into Zanata with your username and password that you used when signup. 
+ 1. You will be redirect to `account inactive` page.
```
Comment (djansen-redhat) [inline docs/user-guide/account/account-resend-activation.md]: *redirected*

File: docs/user-guide/account/account-resend-activation.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
...
+ 1. To resend activation email to the same email address, click on `Resend activation email` button.
+ 1. To update email address and send activation email, type in new email address in the input field and click `Update email`.
```
Comment (djansen-redhat) [inline docs/user-guide/account/account-resend-activation.md]: s/type in/enter the/

File: docs/user-guide/account/account-settings.md [hunk 10]
```diff
@@ -0,0 +1,59 @@
+# Account Settings
+
+To access your user settings:
+
+ 1. Login to Zanata
+ 1. Click on `User menu` on top right corner, click on `Settings` on the drop down menu.
+ 1. Alternately, you can on `Settings` tab in your dashboard page.
```
Comment (djansen-redhat) [inline docs/user-guide/account/account-settings.md]: *Alternatively*, you can *click* on the

**Comment by djansen-redhat** [comment]
Added a spellcheck commit
:+1: Looks good to me

**Comment by seanf** [comment]
I think we should try something like tinypng or pngcrush before taking the step of switching to gifs.

**Comment by aeng** [comment]
Merged into integration/master. https://github.com/zanata/zanata-server/commit/9201cfbb64b32692c801cb9dfedca6a6db20b19f

**Pull request closed without merging**